vendor/src/github.com/docker/notary/pkg/passphrase/passphrase.go

```diff
@@ -86,15 +109,24 @@ func PromptRetrieverWithInOut(in io.Reader, out io.Writer) Retriever {
 		stdin := bufio.NewReader(in)
 
 		indexOfLastSeparator := strings.LastIndex(keyName, string(filepath.Separator))
+		if indexOfLastSeparator == -1 {
+			indexOfLastSeparator = 0
+		}
 
-		if len(keyName) > indexOfLastSeparator+idBytesToDisplay+1 {
-			keyName = keyName[:indexOfLastSeparator+idBytesToDisplay+1]
+		if len(keyName) > indexOfLastSeparator+idBytesToDisplay {
```

Comment:
UX change: show the repository name and 7 digits of the hash of a specific key: `diogomonica/alpine (04fabdf)`